### PR TITLE
HW28.2

### PR DESCRIPTION
**Pull request by pavlokhomiak**

### CODE REVIEW

File: src/main/java/mate/academy/controllers/orders/GetUserOrdersController.java [hunk 1]
```diff
@@ -6,21 +6,23 @@
...
     private static final Injector injector = Injector.getInstance("mate.academy");
     private final OrderService orderService =
             (OrderService) injector.getInstance(OrderService.class);
 
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        List<Order> orderList = orderService.getUserOrders(USER_ID);
+        HttpSession session = req.getSession();
+        Long userId = (Long) session.getAttribute(USER_ID);
```
Comment (OleksiiTromsa) [inline src/main/java/mate/academy/controllers/orders/GetUserOrdersController.java]:
```suggestion
        Long userId = (Long) req.getSession().getAttribute(USER_ID);
```
We use session only once here so maybe we don't need variable for it.

**Comment by OleksiiTromsa** [comment]
Good job

File: src/main/java/mate/academy/controllers/orders/GetUserOrdersController.java (hunk 1 above)
Comment (dmytro-hryhoruk) [reply src/main/java/mate/academy/controllers/orders/GetUserOrdersController.java]: agree

File: src/main/java/mate/academy/controllers/shoppingcart/AddProductToShoppingCartController.java [hunk 2]
```diff
@@ -23,7 +23,9 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        ShoppingCart shoppingCart = shoppingCartService.getByUserId(USER_ID);
+        HttpSession session = req.getSession();
+        Long userId = (Long) session.getAttribute(USER_ID);
+        ShoppingCart shoppingCart = shoppingCartService.getByUserId(userId);
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/controllers/shoppingcart/AddProductToShoppingCartController.java]: same

File: src/main/java/mate/academy/controllers/shoppingcart/CompleteOrderController.java [hunk 3]
```diff
@@ -22,7 +22,9 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        ShoppingCart shoppingCart = shoppingCartService.getByUserId(USER_ID);
+        HttpSession session = req.getSession();
+        Long userId = (Long) session.getAttribute(USER_ID);
+        ShoppingCart shoppingCart = shoppingCartService.getByUserId(userId);
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/controllers/shoppingcart/CompleteOrderController.java]: same

File: src/main/java/mate/academy/controllers/shoppingcart/DeleteProductFromShoppingCartController.java [hunk 4]
```diff
@@ -23,9 +23,11 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
+        HttpSession session = req.getSession();
+        Long userId = (Long) session.getAttribute(USER_ID);
+        ShoppingCart shoppingCart = shoppingCartService.getByUserId(userId);
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/controllers/shoppingcart/DeleteProductFromShoppingCartController.java]: and here

File: src/main/java/mate/academy/controllers/shoppingcart/GetShoppingCartController.java [hunk 5]
```diff
@@ -6,21 +6,23 @@
...
     private final ShoppingCartService shoppingCartService =
             (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
 
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        List<Product> productList = shoppingCartService.getByUserId(USER_ID).getProducts();
+        HttpSession session = req.getSession();
+        Long userId = (Long) session.getAttribute(USER_ID);
+        List<Product> productList = shoppingCartService.getByUserId(userId).getProducts();
```
Comment (dmytro-hryhoruk) [inline src/main/java/mate/academy/controllers/shoppingcart/GetShoppingCartController.java]: same

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**